wid/gatt.py

```diff
@@ -1706,6 +1706,9 @@ def hdl_wid_142(params: WIDParams):
 
     btp.gattc_write(btp.pts_addr_type_get(None), btp.pts_addr_get(None),
                     hdl, '02', 1)
+
+    btp.gattc_write_rsp()
```

Comment:
Could you check if this doesn't break any GATT/CL/GAW tests? Usually, we want to miss the response on purpose because sometimes PTS won't send a response until we respond to WID.

Reply:
Seems like it did not break anything:
```
GATT    GATT/CL/GAW/BI-02-C             PASS
GATT    GATT/CL/GAW/BI-03-C             PASS
GATT    GATT/CL/GAW/BI-04-C             PASS
GATT    GATT/CL/GAW/BI-05-C             PASS
GATT    GATT/CL/GAW/BI-06-C             PASS
GATT    GATT/CL/GAW/BI-07-C             PASS
GATT    GATT/CL/GAW/BI-08-C             PASS
GATT    GATT/CL/GAW/BI-09-C             PASS
GATT    GATT/CL/GAW/BI-11-C             PASS
GATT    GATT/CL/GAW/BI-12-C             PASS
GATT    GATT/CL/GAW/BI-13-C             PASS
GATT    GATT/CL/GAW/BI-32-C             PASS
GATT    GATT/CL/GAW/BI-33-C             PASS
GATT    GATT/CL/GAW/BI-34-C             PASS
GATT    GATT/CL/GAW/BV-02-C             PASS
GATT    GATT/CL/GAW/BV-01-C             PASS
GATT    GATT/CL/GAW/BV-03-C             PASS
GATT    GATT/CL/GAW/BV-05-C             PASS
GATT    GATT/CL/GAW/BV-06-C             PASS
GATT    GATT/CL/GAW/BV-08-C             PASS
GATT    GATT/CL/GAW/BV-09-C             PASS
```